### DISCUSSION TITLE
Indicate shell on first line

### DISCUSSION THREAD

**Discussion by macxcool** [starship/starship#1976] [Q&A] [answered]
I often switch between shells. Is there a way to indicate which shell is active so I can tell at a glance?

**Comment by matchai** [comment, marked as answer]
For sure! Folks have made custom modules that achieve exactly this.
Here's one such example: https://github.com/starship/starship/discussions/1252#discussioncomment-141938

**Reply by macxcool** [reply]
Thanks @matchai. I was able to hack on that and get what I wanted ;-)

```
[custom.printshell]
disabled = false
description = "Show which shell is active"
command = "echo -n $STARSHIP_SHELL"
when = '[ -n "$STARSHIP_SHELL" ]'
shell = ["bash", "--noprofile", "--norc"]
format = "[$output](bg:#AFD700 fg:#111111)[](bg:#464347 fg:#AFD700)"
```
